When downloading from backup page, backup is decrypted (#37324)
home-assistant/home-assistant.github.io: source/_includes/common-tasks/backups.md

### Downloading your local backups

> When downloading the backup from the backup page in Home Assistant, the backup is decrypted on the fly so that you then can view the data using your favorite archive tool. This is done for all backup locations, also when you download from Home Assistant Cloud.

Reviewer: ```suggestion
When downloading the backup from the Home Assistant backup page, it is decrypted on the fly so that you can view the data using your favorite archive tool. This is done for all backup locations and also when you download from Home Assistant Cloud.
```
